tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -39,6 +46,71 @@ public void chown() throws IOException, AlluxioException {
     Assert.assertEquals("user2", owner);
   }
 
+  @Test
+  public void chownUserAndGroup() throws IOException, AlluxioException, Exception {
+    clearLoginUser();
+    GroupMappingServiceTestUtils.resetCache();
+    Configuration.set(PropertyKey.SECURITY_GROUP_MAPPING_CLASS,
+        IdentityUserGroupsMapping.class.getName());
+    Configuration.set(PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.SIMPLE.getAuthName());
```

Comment:
NIT: FYI, `SIMPLE` auth type and `permission.enabled=true` is the default in Alluxio, so you don't need to explicitly set it. But it is okay to leave those here in case we change the defaults in the future :)